cmd/xpumanager_sidecar/README.md

```diff
@@ -76,7 +76,25 @@ master,0.0-1.0_0.1-1.1
 
 ### Use HTTPS with XPU Manager
 
-XPU Manager can be configured to use HTTPS on the metrics interface. For the gunicorn sidecar, cert and key files have to be added to the command:
+There is an alternative deployment that uses HTTPS instead of HTTP. The deployment requires `cert-manager` to provide a certificate for TLS. To deploy:
```

Comment:
perhaps call it a reference/example?
```suggestion
There is an alternative deployment that uses HTTPS instead of HTTP. The reference deployment requires `cert-manager` to provide a certificate for TLS. To deploy:
```